db/schema.rb

```diff
@@ -54,8 +54,6 @@
     t.inet     "last_sign_in_ip"
     t.datetime "created_at",                          null: false
     t.datetime "updated_at",                          null: false
-    t.string   "provider"
```

Comment:
Erin, I now fixed that, but created a new branch "newdrawpieces", because this one wouldn't update the database. I don't know why it happened in the first place, but now it's fixed. :)